### PR TITLE
cp-32: [BUG] Header component is not responsive in Safari browser

### PR DESCRIPTION
**Pull request by juanc-escobar**
The header component was not responsive in the Safari browser, The heather was implemented with flex, so I gave it a max width and fixed the overflows problems for the body, app-container, and body-container since it showed me overflow on Safari. I don't have Mac OS I use an emulator so results may not be 100% accurate but I believe this solved the issue and will prevent future issues for other components (overflow).

### CODE REVIEW

**Review by iamAlinaaa — COMMENTED** [review]
code looks neat )does it work good on your computer?

**Comment by fshabanov** [comment]
Could you, please, share the screenshots?

**Comment by juanc-escobar** [comment]
> Could you, please, share the screenshots? 

Sure! Here are some screenshots at less than 950px in Webkit (Safari) @fshabanov @iamAlinaaa 

image
image
image
image

**Review by OlenaYatseniuk — DISMISSED** [review]

**Review by OlesiaTr — DISMISSED** [review]

**Review by fshabanov — DISMISSED** [review]

**Review by v0ldemar01 — DISMISSED** [review]

**Review by iamAlinaaa — DISMISSED** [review]
looks good

**Review by Nanaier — DISMISSED** [review]

**Comment by juanc-escobar** [comment]
ci Expected — Waiting for status to be reported
ci-pr Expected — Waiting for status to be reported

I updated the branch with the latest changes but had a lot of trouble with docker and node modules so I had to delete and reinstall, it works in my local environment and I did the push but the ci and ci-pr won't change status.

**Comment by liza-veis** [comment]
you need to fix conflicts to make it work

**Comment by juanc-escobar** [comment]
@Liza-Veis 

I reverted the changes and installed @esbuild/win32-x64 as dependency, also pulled the latest changes in but didn't have a merge conflict. 

now it's showing me this in the PR:

Conflicting files
frontend/src/libs/components/app/styles.module.scss
package-lock.json

but does not show me the conflict and vs. code after pulling the latest changes didn't show me conflict how should I solve this?

**Comment by liza-veis** [comment]
which command did you use to pull new changes?

**Comment by juanc-escobar** [comment]
@Liza-Veis

git pull origin fix/cp-32-header-component

**Comment by fshabanov** [comment]
> @Liza-Veis
> 
> git pull origin fix/cp-32-header-component

@juanc-escobar you should use `git pull origin development`

**Comment by juanc-escobar** [comment]
thanks for the clarification. @fshabanov @Liza-Veis 

I was able to successfully merge the latest changes.

**Review by BrianSammit — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "@typescript-eslint/parser": "6.4.0",
     "danger": "11.2.8",
     "editorconfig-checker": "5.1.1",
+    "esbuild": "0.19.2",
```
Comment (liza-veis) [inline package.json]: if you remove node_modules and reinstall them again this should work without this dependency, maybe we will add it later but only for frontend
```suggestion
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Comment by juanc-escobar** [comment]
@Liza-Veis  I uninstalled esbuild ✅

**Review by liza-veis — APPROVED** [review]

**Review by fshabanov — APPROVED** [review]

**Pull request merged**